in_honor_donation.html

```diff
@@ -1,10 +1,35 @@
 <fieldset id="in-honor-of" class="ak-donate-area-step-1 form-style-labelabove">
     <div class="input checkbox input-checkbox margin-top-small margin-bottom-none text-strong">
       <input type="checkbox" value="1" name="inhonorcheck" id="in-honor-check" />
+      {% if page.custom_fields.custom_donate_honor_label %}
+      <label for="in-honor-check">{{ page.custom_fields.custom_donate_honor_label }}</label>
+      {% else %}
       <label for="in-honor-check">{% filter ak_text:"donate_inhonorof_option_label" %}Donate in
         someone's honor.{% endfilter %}</label>
+      {% endif %}
     </div>
     <div id="in-honor-of-fields" class="margin-bottom-normal">
+      {% if page.custom_fields.donate_honoree_alt_spelling == "Y" %}
+      <div class="input text input-text ak-">
```

Comment:
Noted an incomplete classname with the trailing hyphen `'ak-' `here. Not sure if this is intentional or does it need cleanup?